.gitmodules

```diff
@@ -128,6 +128,9 @@
 [submodule "contrib/icu"]
 	path = contrib/icu
 	url = https://github.com/unicode-org/icu.git
+[submodule "contrib/flatbuffers"]
+	path = contrib/flatbuffers
+	url = https://github.com/google/flatbuffers.git
```

Comment:
Note that Flatbuffers is no longer a build dependency of Apache Arrow in C++ (we have vendored the necessary files)